src/openrct2/scripting/ScEntity.hpp

```diff
@@ -56,8 +56,9 @@ namespace OpenRCT2::Scripting
                     case EntityType::Vehicle:
                         return "car";
                     case EntityType::Guest:
+                        return "guest";
                     case EntityType::Staff:
-                        return "peep";
+                        return "staff";
```

Comment:
As discussed in discord#plugin, we will probably now have an `targetApiVersion` in the plugin header that controls the behaviour.